test/jsToAqua.test.ts

```diff
@@ -14,58 +14,38 @@
  * limitations under the License.
  */
 
-import { jsToAquaImpl } from "../src/lib/helpers/jsToAqua.js";
+import { jsToAquaImpl, makeOptional } from "../src/lib/helpers/jsToAqua.js";
 
 const WRAPPER_FUNCTION_NAME = "res";
-const RETURN_TYPE = "Res";
 
 const obj = [
```

Comment:
What is it? How is it used?

Reply:
moved this inside tests so it's more clear how it's used